<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up | GE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 64, 115);</string>
  </property>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgb(59, 109, 188);
color: rgb(125, 169, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string> Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpass">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgb(59, 109, 188);
color: rgb(125, 169, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string> Confirm Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:14pt &quot;Tahoma&quot;;
background-color: rgb(55, 100, 173);
color: rgb(5, 26, 59);
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="message1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>510</y>
     <width>411</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Tahoma&quot;;
color: rgb(231, 35, 35);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>add.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgb(59, 109, 188);
color: rgb(125, 169, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string> Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgb(59, 109, 188);
color: rgb(125, 169, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string> Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Tahoma&quot;;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgb(59, 109, 188);
color: rgb(125, 169, 240);
padding-bottom:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="placeholderText">
    <string> Surname</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
